packages/contractkit/src/identity/claims/account.test.ts

```diff
@@ -68,7 +68,8 @@ testWithGanache('Account claims', (web3) => {
       IdentityMetadataWrapper.fetchFromURL = () => Promise.resolve(otherMetadata)
 
       const metadata = IdentityMetadataWrapper.fromEmpty(address)
-      signedClaim = await metadata.addClaim(
+      claim = createAccountClaim(otherAddress)
+      await metadata.addClaim(
```

Comment:
Not sure what’s happening here? Did you mean to pass in claim?

Reply:
I did!